src/SprykerEco/Zed/Payone/Business/Payment/DataMapper/StandartParameterMapper.php

```diff
@@ -45,7 +45,7 @@ public function setStandardParameter(AbstractRequestContainer $container, Payone
     {
         $container->setApiVersion(PayoneApiConstants::API_VERSION_3_9);
         $container->setEncoding($standardParameter->getEncoding());
-        $container->setKey($this->hashGenerator->hash($standardParameter->getKey()));
+        $container->setKey($this->hashGenerator->hash($standardParameter->getKey() ?? ''));
```

Comment:
wrong
probably `if` can do a trick
there is no reason to pass empty string everywhere

Reply:
Fixed.